README.md

```diff
@@ -26,6 +26,12 @@ The following packages are meant for direct consumption by Instana users, all  o
 * `@instana/collector`: See <https://www.ibm.com/docs/de/obi/current?topic=technologies-monitoring-nodejs>.
 * `@instana/google-cloud-run`: See <https://www.ibm.com/docs/de/obi/current?topic=agents-monitoring-google-cloud-run#instana-cloud-native-buildpack-for-google-cloud-run> or <https://www.ibm.com/docs/de/obi/current?topic=agents-monitoring-google-cloud-run#nodejs>.
 
+## Breaking Changes
+
+The current major release is `2.x.y`.
+
+Checkout our [breaking changes documentation](https://www.ibm.com/docs/de/obi/current?topic=technologies-monitoring-nodejs#supported-nodejs-versions#breaking-changes) to upgrade from v1 to v2.
```

Comment:
Wasn't sure about the link structure because it's not live yet

Reply:
```suggestion
Checkout our [breaking changes documentation](https://www.ibm.com/docs/en/obi/current?topic=technologies-monitoring-nodejs#breaking-changes) to upgrade from v1 to v2.
```

Anchors are not hierarchical, so this ^ should be correct.